### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by PowerLevel9000**
# worked on ui of the calculator

- [ ] dynamic buttons
- [ ] components
- [ ] nested components

## happy coding 

WannaCodeCodeGIF

### CODE REVIEW

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @PowerLevel9000, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly fix the MIT broken link in your README file 😉 
- [ ] Also please kindly delete all the boilerplate that got created by the CRA in the `public` file except the `index.html` file, you didn’t delete all 🙃 🙃

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [OPTIONAL] I notice that your commit messages are not capitalized🙃 . The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Please don't forget this for your next commit messages 🙃

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PowerLevel9000** [comment]
hi @cynthiainga thank for reviewing my code 

I definitely fix those required changes

but I want to draw your attention about this mit license link https://github.com/PowerLevel/mathmatica/blob/dev/LICENSE
✅ this link for future as I will merge this in dev 
✅  but for now this link is this https://github.com/PowerLevel/mathmatica/blob/calculatorUi/LICENSE

please consider these things

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,77 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  background: url('./media/math-background.jpg');
+  width: 100%;
+  height: 100%;
+}
```
Comment (t-yanick) [inline src/index.css:11]: [ OPTIONAL]
- Project requiements asks for our Calculator.js component to match given design and I think matching designs always incude colors and typograpgy. Check screenshots below to see difference.
- Given design
Screen Shot 2023-02-21 at 6 34 37 PM
- Your design
Screen Shot 2023-02-21 at 6 34 56 PM

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @PowerLevel9000 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All boilerplate generated by CRA deleted
:heavy_check_mark: Use of functional components

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions

- [OPTIONAL ] Previous reviewer suggested you capitalize your commit messages, but your last commit messages afrer his proposal does not meet this suggestion. Please kindly ensure you capitalize your commit messages henceforth.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (PowerLevel9000) [reply src/index.css:11]: @t-yanick 
hi Cr, I hope you are doing well

as I am making this my own project I like to implement my own design well I like  to switch to the given design for approval and change it to design what I like after approval and as sir, you marked it optional sorry to say that I won't change thanks for the review
sorry to say that it is not feasible according to my thoughts

**Pull request merged**